### ISSUE TITLE
So many temperature choices, which is "best"?  A max value perhaps?

### ISSUE THREAD

**Issue by jkleckner** [yujitach/MenuMeters#273]
I really like seeing the temperature of my processor in MenuMeters.
That said, there are a huge list of choices [1].
Which of these is the "best" choice?  (I would love to get people's opinions)

Would it make sense to add a meta measure as is done with the "average" CPU for temperature such as the "max" across all sensors?

Thank you so much for keeping this great tool alive!

[1] <img width="554" alt="MenuMeters-TemperatureChoices" src="https://user-images.githubusercontent.com/1223485/143301833-be36f5da-4a60-45ed-9ff9-6038e356398f.png">

**Comment by srcshelton** [comment]
This might be relevant:

https://twitter.com/marcan42/status/1484158082620203008

**Comment by jkleckner** [comment]
Yes, interesting.  I still would sort of like a "Max Temp" if it is practical.

**Comment by victorlin** [comment]
> https://twitter.com/marcan42/status/1484158082620203008

The link is broken. I extracted relevant images [from archive.org](https://web.archive.org/web/20221124160232/https://twitter.com/marcan42/status/1484158082620203008):

<details>
<summary>Expand</summary>

![1](https://github.com/yujitach/MenuMeters/assets/13424970/c68afe43-7c59-4505-8ac9-a38778eb459e)
![3](https://github.com/yujitach/MenuMeters/assets/13424970/20b874e6-25fb-46ab-acf8-92981a9c6b66)
![4](https://github.com/yujitach/MenuMeters/assets/13424970/61996669-4b42-47d8-80ee-a7b93256cbf8)

</details>
